### DISCUSSION TITLE
Buying Premium key without paypal

### DISCUSSION THREAD

**Discussion by lavakalas** [mtotschnig/MyExpenses#1575] [Q&A]
Hi! I live in Russia and is really fond of this app. Would like to buy a subscription to a professional key, but unfortunately PayPal doesn't work in Russia at all. Is there any other way I can purchase premium key in rubles?

**Comment by mtotschnig** [comment]
https://www.myexpenses.mobi/en/#participate mentions the option to pay with Bitcoin. Please contact support@myexpenses.mobi if you have further questions.
